src/main/arm/createUiDefinition.json

```diff
@@ -9,28 +9,28 @@
                 "type": "Microsoft.Common.InfoBox",
                 "options": {
                     "icon": "Info",
-                    "text": "Provide an entitled IBMid to provision the WebSphere cluster. <a href='https://www.ibm.com/ibmid/myibm/help/us/helpdesk.html' target='_blank'>Learn more</a>"
+                    "text": "This offer is Bring-Your-Own-License. You will be required to enter your registered IBM ID prior to successfully deploying this offer and your IBM ID must have active WebSphere entitlements associated with it. If you find that provisioning fails due to lack of entitlements, please contact the primary or secondary contacts for your IBM Passport Advantage Site who should be able to grant you access or follow steps at <a href='https://www-112.ibm.com/software/howtobuy/passportadvantage/homepage/ecarec' target='_blank'>IBM eCustomer Care</a> for further assistance. This offer also assumes you are properly licensed to run offers in Microsoft Azure."
```

Comment:
I have requested our documentation team to review this message.

Also, I am not sure what this means: `This offer also assumes you are properly licensed to run offers in Microsoft Azure.`? How can a customer check that?

Reply:
@git4rk  I copied message from #40, which is provided by @gcharters . Ranjan, could you pls work with Graham to figure out the final version of the message?

Reply:
@majguo Here is the message from our documentation team:

This offer is Bring-Your-Own-License. To deploy this offer, you must enter your registered IBMid and your IBMid must have active WebSphere entitlements associated with it. If provisioning fails due to lack of entitlements, ask the primary or secondary contacts for your IBM Passport Advantage site to grant you access or follow steps at [IBM eCustomer Care](https://www-112.ibm.com/software/howtobuy/passportadvantage/homepage/ecarec) for further assistance. This offer also assumes you are properly licensed to run offers in Microsoft Azure.

Reply:
Thanks. Fixed in b6c4aa8.